source/models/defining-models.md

```diff
@@ -180,6 +174,18 @@ export default DS.Model.extend({
 });
 ```
 
+#### Synchronous relationships
+
+Starting with Ember Data 2.0 all relationships are asynchronous by default. If
```

Comment:
my thing here is that it's a breaking update, and one that might go unnoticed if not mentioned explicitly. I think I'd rather add the explanation then.